modules/phase_field/src/userobjects/ComputeGrainForceAndTorque.C

```diff
@@ -78,7 +78,7 @@ ComputeGrainForceAndTorque::execute()
         for (_qp=0; _qp<_qrule->n_points(); ++_qp)
           if (_dF[_qp][j](0) != 0.0 || _dF[_qp][j](1) != 0.0 || _dF[_qp][j](2) != 0.0)
           {
-            const RealGradient compute_torque =_JxW[_qp] * _coord[_qp] * (_q_point[_qp] - centroid).cross(_dF[_qp][j]);
+            const RealGradient compute_torque =_JxW[_qp] * _coord[_qp] * (_current_elem->centroid() - centroid).cross(_dF[_qp][j]);
```

Comment:
This helps in optimization while calculating the velocities later. Change in this userobject is done just to keep the formulation consistent everywhere.